fbgemm_gpu/src/sparse_ops.cu

```diff
@@ -2128,6 +2132,52 @@ Tensor permute102_baddbmm_permute102_cuda(
   // C (m, b, n) = A (m, b, k) * B (b, k, n) ---> row major
   // C (m, b, n) = (B^T (b, k, n) * A^T (m, b, k))^T ---> column major
 
+#ifdef __HIP_PLATFORM_HCC__
+  float alpha = 1.0f;
+  float beta = 1.0f;
+
+  auto Btype = HIPBLAS_R_16F;
+  auto ldb = n;
+  auto strideB = n * k;
+
+  auto Atype = HIPBLAS_R_16F;
+  auto lda = k * batch_size;
+  auto strideA = k;
+
+  auto Ctype = HIPBLAS_R_16F;
+  auto ldc = n * batch_size;
+  auto strideC = n;
+
+  auto computeType = HIPBLAS_R_32F;
+
+  auto result = hipblasGemmStridedBatchedEx(
```

Comment:
Just out of curiosity: why only this function is supported by hipBLAS but not rocBLAS?